tests/publiccloud/slem_basic.pm

```diff
@@ -56,20 +62,19 @@ sub run {
 
     select_serial_terminal();
 
-    if ($qam) {
+    if ($qam or $args->{my_instance}) {
         $instance = $self->{my_instance} = $args->{my_instance};
         $provider = $self->{provider} = $args->{my_provider};
     } else {
         $provider = $self->provider_factory();
         $instance = $self->{my_instance} = $provider->create_instance(check_guestregister => 0);
     }
-    $provider->{username} = 'suse';
```

Comment:
This line removed being `user` managed by `PUBLIC_CLOUD_USER` setting, that when empty `default` user applied.

Reply:
Removed in PR https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19186